Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/AcpiPlatform.c

```diff
@@ -361,7 +361,7 @@ PlatformUpdateAcpiTable (
   } else if (Table->Signature == SIGNATURE_32 ('R', 'T', 'C', 'T')) {
     DEBUG ((DEBUG_INFO, "Find RTCT table\n"));
 
-#if FixedPcdGet8 (PcdTccEnabled) && !defined(PLATFORM_RPLS) && !defined(PLATFORM_ADLN) && !defined(PLATFORM_ASL)
+#if FixedPcdGet8 (PcdTccEnabled) && !defined(PLATFORM_RPLS) && !defined(PLATFORM_BTLS) && !defined(PLATFORM_ADLN) && !defined(PLATFORM_ASL)
```

Comment:
RTCT table should be removed from ACPI table, and remove this code. maybe this could be done in a separate PR.